app/controllers/users_controller.rb

```diff
@@ -145,12 +145,11 @@ def standing
     @lookup = rows.to_h
 
     @flagged_comments = @showing_user.comments
-      .where("
-        comments.flags > 0 and
-        comments.created_at >= now() - interval #{@interval[:dur]} #{@interval[:intv]}")
-      .order("id DESC")
-      .includes(:user, :hat, story: :user)
+      .where("comments.flags > ?", 0)
+      .where("comments.created_at >= ?", "now() - interval #{@interval[:dur]} #{@interval[:intv]}")
```

Comment:
It would be nice to move this into AR but I don't know if it's expressive enough for us to do it.